.travis.yml

```diff
@@ -3,9 +3,11 @@ python:
     - "2.7"
 install:
     - pip install requests junit_xml beaker-client mock
-    - pip install pylint pycodestyle
+    - pip install pylint pycodestyle flake8
     - pip install .
+before_script:
+    - flake8 --show-source .
+    - pycodestyle .
```

Comment:
Travis docs [say](https://docs.travis-ci.com/user/customizing-the-build/#Breaking-the-Build) that any failure in `before_script` aborts the whole "build", i.e. none of the `script` commands would run. Was that your intention?